docs/monitoring/monitoring-models.ipynb

```diff
@@ -0,0 +1,317 @@
+{
```

Comment:
This notebook is very hard to follow, I don't see how to get to those screens and what's the context for it. Need to have it as part of the flow
Also, for the dashboards, are these the same for the new infrastructure?

Reply:
@gilad-shaham It's an overview. Specific graphs are mentioned in the user-flow/tutorial. Dashboards are updated. (This page was pre-existing, I just updated.)